### PR TITLE
Angular2 : Fix imports and upgrade audits service

### PR DESCRIPTION
**Pull request by wmarques**
Fixed the imports in admin module (the `./` was missing).
Also upgraded the audits services following official guide, it should work if we put the correct model in the interface but I have no idea about the json returned...

### CODE REVIEW

File: generators/client-2/templates/src/main/webapp/app/_app.main.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import { upgradeAdapter } from './upgrade_adapter';
 import { HTTP_PROVIDERS } from '@angular/http';
+import { AuditsService } from './admin/audits/audits.service';
 import './app.module';
 
 upgradeAdapter.bootstrap(document.body, ['<%=angularAppName%>.app'], {strictDi: true});
+upgradeAdapter.addProvider(AuditsService);
 upgradeAdapter.addProvider(HTTP_PROVIDERS);
+
+angular.module('<%=angularAppName%>.admin')
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/_app.main.ts]: why is this not registered in the `admin.module.ts`? any specific reason

Comment (wmarques) [inline generators/client-2/templates/src/main/webapp/app/_app.main.ts]: Oh yes I should have done it there sorry, I'm fixing this

File: generators/client-2/templates/src/main/webapp/app/_app.main.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { upgradeAdapter } from './upgrade_adapter';
 import { HTTP_PROVIDERS } from '@angular/http';
+import { AuditsService } from './admin/audits/audits.service';
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/_app.main.ts]: can you remove the import here?

File: generators/client-2/templates/src/main/webapp/app/admin/audits/_audits.service.ts [hunk 3]
```diff
@@ -1,20 +1,21 @@
 import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+import 'rxjs/add/operator/map';
 
-AuditsService.$inject = ['$resource'];
+export interface AuditsData {
+    //TODO put the model returned here
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/admin/audits/_audits.service.ts]: are you gonna put the modal here?

Comment (sendilkumarn) [inline generators/client-2/templates/src/main/webapp/app/admin/audits/_audits.service.ts]: I think no modal here

Comment (wmarques) [inline generators/client-2/templates/src/main/webapp/app/admin/audits/_audits.service.ts]: I have no idea of the model (not the modal) returned by the backend... I'll check this in java code

**Pull request merged**